README.md

```diff
@@ -186,3 +186,17 @@ Requires `GITLAB_ACCESS_TOKEN` variable to be set, which is an access token with
   - `SSH_PRIVATE_KEY` A private key of a user which can push to Pantheon
   - `SSH_KNOWN_HOSTS` The result of running `ssh-keyscan -H codeserver.dev.$PANTHEON_SITE_ID.drush.in`
   - `TERMINUS_PLUGINS` Comma-separated list of Terminus plugins to be available (optional)
+
+### composer-lock-diff comment on PR updates
+
+- Add the following the composer.json to enable composer-lock-diff comments on your PR/repository.
```

Comment:
How about instead of
"Add the following the composer.json to enable composer-lock-diff comments on your PR/repository."
...
"Add the following to your repository's composer.json to enable composer-lock-diff comments on your PRs."

Reply:
Much better :)